### PR TITLE
fix: adds I18nT keyword into key collection

### PR DESCRIPTION
**Pull request by advisciglio**
fix: https://github.com/intlify/eslint-plugin-vue-i18n/issues/447

### CODE REVIEW

File: lib/utils/collect-keys.ts [hunk 1]
```diff
@@ -210,13 +210,11 @@ export function collectKeysFromAST(
         }
       } else {
         if (
-          (node.key.name === 'path' &&
-            (node.parent.parent.name === 'i18n' ||
-              node.parent.parent.name === 'i18n-t')) ||
-          (node.key.name === 'keypath' && node.parent.parent.name === 'i18n-t')
+          (node.key.name === 'path' && (['i18n', 'i18n-t', 'i18nt'].includes(node.parent.parent.name)) ||
+          (node.key.name === 'keypath' && ['i18n-t', 'i18nt'].includes(node.parent.parent.name)))
```
Comment (ota-meshi) [inline lib/utils/collect-keys.ts:214]: Could you please use rawName to determine more accurately?

```suggestion
          (node.key.name === 'path' &&
            (node.parent.parent.name === 'i18n' ||
              node.parent.parent.name === 'i18n-t' ||
              node.parent.parent.rawName === 'I18nT')) ||
          (node.key.name === 'keypath' &&
            (node.parent.parent.name === 'i18n-t' ||
              node.parent.parent.rawName === 'I18nT'))
```

**Review by ota-meshi — CHANGES_REQUESTED** [review]
Thank you for this PR!

File: lib/utils/collect-keys.ts (hunk 1 above)
Comment (advisciglio) [reply lib/utils/collect-keys.ts]: Thanks for the suggestion, I've fixed it accordingly.

**Review by ota-meshi — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**